config/locales/client.en.yml

```diff
@@ -52,6 +52,9 @@ en:
       download_csv: "CSV"
       others_dirty: "A query has unsaved changes that will be lost if you navigate away."
       run_time: "Query completed in {{value}} ms."
+      single_result_count: "{{value}} result."
+      total_result_count: "{{value}} results."
```

Comment:
You should use the special `%{count}` variable when you don't know the exact number. See https://guides.rubyonrails.org/i18n.html#pluralization for more on that.

I'd suggest you use the following YML instead:
```yml
result_count:
  one: "%{count} result."
  other: "%{count} results."
```